web-common/src/features/sources/add-source/RemoteSource.svelte

```diff
@@ -123,6 +142,23 @@
               false
             );
           }
+          if ($createSourceMutation.isError || error) {
```

Comment:
Get this error based on file path, including updating the error shown to the user.